### DISCUSSION TITLE
How to prevent setting of the siren accidentally

### DISCUSSION THREAD

**Discussion by fpaddict** [dgreif/ring#1088] [Q&A]
Hi all, one of my cameras is in the “Patio” location which is where i also have lights. Typically I say “Siri, turn on the patio lights” but today, I just said “Siri, turn on the patio”. This caused the lights and the siren of the ring camera to go off. 

Is there a setting to disable the siren from HomeKit so that it doesn’t get triggered again?

**Comment by dgreif** [comment]
If you want to completely remove the siren from HomeKit, you can use the `hideCameraSirenSwitch` setting
